docs/crashlytics/usage/index.md

```diff
@@ -233,3 +233,37 @@ React Native Crashlytics module is generating additional non-fatal issues on Jav
   }
 }
 ```
+
+## Crashlytics non-fatal exceptions native handling
+
+In case you need to log non-fatal (handled) exceptions on the native side (e.g from `try catch` block), you may use the following static methods:
+<br />
+### Android
+
+```
+try {
+  //...
+} catch (Exception e) {
+  ReactNativeFirebaseCrashlyticsNativeHelper.recordNativeException(e);
```

Comment:
Interesting - now that I think about it, I wonder whether this is necessary? If it's not using anything react-native-firebase specific, is this really any more or less convenient then just using the API directly like this?

```suggestion
  FirebaseCrashlytics.getInstance().recordException(e);
```

Reply:
So couple concerns that lead me to the code above, I might be missing out on some stuff, so please correct me if I'm wrong:
 * FirebaseCrashlytics class is not exposed to the native code unless we add an app-level Gradle dependency for it;
 * If we add that dependency, will it still be the same instance in the native code vs `react-native-firebase` code?;
 * If we add a dependency, and it's instance does not overlap with `react-native-firebase`'s one - do we have to keep an eye on crashlytics version in `react-native-firebase` and match it in the app-level Gradle to keep it so?.<br />

Thank you.

Reply:
Great point, I can see that being a maintenance headache, I accept the defense

gradle will make sure to resolve only one version, and it should be static so no problem with getting the same actual running instance, however, you would still need to do the equivalent of a manual link (like before react-native had auto-linking) which is painful